README.md

```diff
@@ -2,3 +2,27 @@
 
 [![Join the chat at https://gitter.im/hitraffic/mobile](https://badges.gitter.im/Join%20Chat.svg)](https://gitter.im/hitraffic/mobile?utm_source=badge&utm_medium=badge&utm_campaign=pr-badge&utm_content=badge)
 A hybrid mobile app powered by the Ionic Framework.  Utilizes the hitraffic API to provide traffic incident reports.
+
+---
+
+Installation
+-----------------------------
+
+1. Fork and clone this repo
+2. Run `npm install`
+3. Run `bower install`
+4. Run `ionic lib update` and enter `yes` when prompted
+
```

Comment:
Getting tripped up on some small things. Should probably mention that you need to install Ionic _and_ Cordova globally with `npm install -g ionic cordova`